electrical/motor_controller.py

```diff
@@ -1,20 +1,22 @@
 import time
 #from engine.robot import Robot
-if False:  # change to True when running code on robot
-    import RPi.GPIO as GPIO
-    
+import RPi.GPIO as GPIO
+
 class BasicMotorController:
     """ 
     BasicMotorController contains pinouts to configure motor controller, as well as 
     commands to physically move the robot. 
     """
-    def __init__(self, robot):
+
+    def __init__(self):
         # raspberry pi motor driver pinouts
         self.in1 = 5
         self.in2 = 6
-        self.enA = 13   #PWM
-        self.enB = 12   #PWM 
-        self.is_sim = robot.is_sim
```

Comment:
what's the reasoning for deleting this?

Reply:
I believe many of these changes were done by Jerry or another electrical member before I started working on the motor_test.py file. It's under my name though and on this pull request because the raspberry pi's local files were never pushed to github until I logged in with my account and pushed them (so unfortunately I can't really explain the reasoning for some of the changes like this one).